src/components/Tabs/Annotation/CreateLabelForm.tsx

```diff
@@ -36,6 +36,10 @@ const labelTypeToDisplayName: Record<LabelType, string> = {
   [LabelType.CUSTOM]: "Custom",
 };
 
+const isMetadataColumnName = (name: string): boolean => {
+  return name === CSV_COL_ID || name === CSV_COL_TIME || name === CSV_COL_TRACK;
+};
```

Comment:
Added validation to ensure users can't make labels with metadata names (or blank labels)